app/src/main/res/layout/item_reward.xml

```diff
@@ -153,6 +170,7 @@
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_marginTop="@dimen/grid_3"
+          android:focusable="false"
```

Comment:
Curious why we're setting this focusable to false!

Reply:
This is the placeholder invisible button that I use to give the bottom of the reward card the right padding!